### ISSUE TITLE
Confusion of where to enter text in textfields

### ISSUE THREAD

**Issue by pauljanes** [krico/jas#289]
In the old release it was clearer where text needed to be entered. It was in a box below the label. But in the newer version it is not so clear as the text that was a label is now a placeholder. So I keep trying to enter text below what I think is the label, but actually the text needs to be entered where the placeholder is. We need to make it very clear where the text should be entered on the screen, and which are merely labels. Text boxes should have margins all around them. At the moment it appears they only have a bottom margin. So do I click above or below this margin is the question if I am a new user to the system.

**Comment by krico** [comment]
Paul, I think what you meant to say was "... Text inputs should have
BORDERS around them..."  Right?

**Comment by pauljanes** [comment]
Yes any input boxes should have borders around them, as I thought they were labels, but actually they were input boxes. So I was clicking in wrong place

**Comment by dragonflypl** [comment]
Hi,

whole GUI is based on Material Design approach: https://material.angularjs.org/#/demo/material.components.input

But of course we can change it. I'll play around with borders.

**Comment by krico** [comment]
Paul, are you referring to ALL the forms, or is there any form in particular where you had this problem.

I had a second look at some of the forms and they seem very intuitive to me.  I looked at other sites and it seems to be common practice...  Maybe rather than borders we should have some kind of "hover" thing.

Google mail for example:

![image](https://cloud.githubusercontent.com/assets/6952185/7551775/c746043a-f695-11e4-9f53-06830f9c5f68.png)

This is ours:

![image](https://cloud.githubusercontent.com/assets/6952185/7551765/7c72d9f6-f695-11e4-8486-942305c3648d.png)

![image](https://cloud.githubusercontent.com/assets/6952185/7551768/8bde80de-f695-11e4-91d3-849b52b58e05.png)

**Comment by pauljanes** [comment]
Maybe its because we changed it and I was used to the old way where there was a label and then a textbox.

Lets see how users react if you like. Hillary tried it and she clicked the right place.

Maybe Chris try with Carol and Waldemar with Aga. If all 3 wifes pass the test we can leave it as it is

**Comment by pauljanes** [comment]
Hover sounds like a good idea by the way. Like in the demo that Pawel sent earlier

**Comment by krico** [comment]
I just did the test with Carol (my wife) and she failed :-(

She clicked below the "placeholder" and started typing, then when text didn't show up she said "It's not working"

@wszarmach sent a nice e-mail with suggestions, I like the 2nd image with the triangle on the right of the bottom border, and I quote:
1. Maybe putting an icon to the left of the entry field
![image](https://cloud.githubusercontent.com/assets/6952185/7551860/6e2a8f80-f698-11e4-98d9-c72167c0bafa.png)
2. Always having the title above and a hint text in the entry field
![image](https://cloud.githubusercontent.com/assets/6952185/7551862/84c66750-f698-11e4-9bf8-0feceabc2dde.png)
3. Title left of entry field with hint text in entry field
![image](https://cloud.githubusercontent.com/assets/6952185/7551864/9817ac2e-f698-11e4-8e5a-09b0d7455a8b.png)

**Comment by pauljanes** [comment]
Tell Carol she was not the only one :-) I did exactly same as her.

I like Waldemars suggestions. 1, 2 and 3 are all fine by me. But considering Carol and I both failed, it seems like we need to do something
